playground/app.js

```diff
@@ -176,6 +211,7 @@ class App extends Component {
               formData={this.state.formData}
               onChange={data => this.setState({formData: data.formData})}
               onSubmit={data => this.setState({formData: data.formData})}
+              fields={{geo: GeoPosition}}
```

Comment:
Well, that's part of the example from the doc, and furthermore the registration is approached in the section before... Well, I'll see if I can improve it then.